home/gettingstarted/0.7/persistors.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docsdefault07
+title: Persistors
+permalink: /home/gettingstarted/0.7/persistors/index.html
+
+smversion: 0.7
+partof: getting-started
+num: 6
+outof: 50
+---
+
+
+`Persistor`s are used to store results from executed benchmarks.
```

Comment:
Add: "They can be specified by overriding the `persistor` method in your benchmark class.